src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,26 @@
+// import renderer from 'react-test-renderer';
+import calculate from '../logic/calculate';
```

Comment:
- [ ] Kindly test all functionality and scenarios for the `calculate` file as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Write%20unit%20tests%20for%20the%20files%20operate.js%20and%20calculate.js%20using%20Jest.).

Reply:
Hi! it's done!

Reply:
Hi @lmunozm1702 you only test the `AC` operations. You must test all other operations and scenarios.

Reply:
I'm sorry!!! I write the comment but forget the commit and push, now is really done!

Reply:
Ok, No problem.